src/icon/Icons.tsx

```diff
@@ -1256,6 +1256,16 @@ export const Globe = () => (
 );
 
 //H
+export const HardDriveFilled = () => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fillRule="evenodd" clipRule="evenodd" d="M17.447 4.658L20.789 11.342C20.927 11.618 21 11.928 21 12.236V18C21 19.654 19.654 21 18 21H6C4.346 21 3 19.654 3 18V12.236C3 11.928 3.073 11.618 3.211 11.342L6.553 4.658C7.064 3.636 8.092 3 9.236 3H14.764C15.908 3 16.935 3.636 17.447 4.658ZM8.342 5.553L5.618 11H18.382L15.658 5.553C15.488 5.212 15.145 5 14.764 5H9.236C8.855 5 8.512 5.212 8.342 5.553ZM12 17H16C16.55 17 17 16.55 17 16C17 15.45 16.55 15 16 15H12C11.45 15 11 15.45 11 16C11 16.55 11.45 17 12 17ZM8 17C7.45 17 7 16.55 7 16C7 15.45 7.45 15 8 15C8.55 15 9 15.45 9 16C9 16.55 8.55 17 8 17Z" fill="white" fillOpacity="0.9"/>
+    <mask id="mask0_1784_33484" style={{maskType:"luminance"}} maskUnits="userSpaceOnUse" x="3" y="3" width="18" height="18">
+      <path fillRule="evenodd" clipRule="evenodd" d="M17.447 4.658L20.789 11.342C20.927 11.618 21 11.928 21 12.236V18C21 19.654 19.654 21 18 21H6C4.346 21 3 19.654 3 18V12.236C3 11.928 3.073 11.618 3.211 11.342L6.553 4.658C7.064 3.636 8.092 3 9.236 3H14.764C15.908 3 16.935 3.636 17.447 4.658ZM8.342 5.553L5.618 11H18.382L15.658 5.553C15.488 5.212 15.145 5 14.764 5H9.236C8.855 5 8.512 5.212 8.342 5.553ZM12 17H16C16.55 17 17 16.55 17 16C17 15.45 16.55 15 16 15H12C11.45 15 11 15.45 11 16C11 16.55 11.45 17 12 17ZM8 17C7.45 17 7 16.55 7 16C7 15.45 7.45 15 8 15C8.55 15 9 15.45 9 16C9 16.55 8.55 17 8 17Z" fill="white"/>
+    </mask>
+    <g mask="url(#mask0_1784_33484)"></g>
```

Comment:
I think this mask is an artifact from the figma but not part of the actual svg. can you pull from the actual source? https://akveo.github.io/eva-icons/#/?searchKey=hard&type=outline